corehq/apps/userreports/tests/test_expressions.py

```diff
@@ -807,3 +810,33 @@ def test_unsupported_evluator_statements(self, eq, context):
         "context_variables": context
     })
     self.assertEqual(expression({}), None)
+
+
+class TestFormsExpressionSpec(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.domain = uuid.uuid4().hex
+        factory = CaseFactory(domain=cls.domain)
+        [cls.case] = factory.create_or_update_case(CaseStructure())
+        cls.forms = [f.to_json() for f in FormAccessors(cls.domain).get_forms(cls.case.xform_ids)]
+
+        cls.expression = ExpressionFactory.from_spec({
+            "type": "get_case_forms",
+            "case_id_expression": {
+                "type": "property_name",
+                "property_name": "_id"
+            },
+        })
+
+    def test_evaluation(self):
```

Comment:
would be good practice to add the `@run_with_all_backends` decorator to this and make sure it works with both couch and sql